Tools/src/FormatCheckTool/FormatCheckTool.cs

```diff
@@ -0,0 +1,234 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Text;
+using Microsoft.Recognizers.Text;
+using Microsoft.Recognizers.Text.Choice;
+using Microsoft.Recognizers.Text.DateTime;
+using Microsoft.Recognizers.Text.Number;
+using Microsoft.Recognizers.Text.NumberWithUnit;
+using Microsoft.Recognizers.Text.Sequence;
+using Newtonsoft.Json;
+using Microsoft.Recognizers.Text.DataDrivenTests;
+
+namespace FormatCheckTool
+{
+    class FormatCheckTool
+    {
+        private static readonly String PleaseCheckStr = "Please Check it.";
+
+        static void Main(string[] args)
+        {
+            // Encoding for 'exotic' characters e.g. '€'
+            Encoding.RegisterProvider(CodePagesEncodingProvider.Instance);
+
+            ShowIntro();
+
+            while (true)
+            {
+                Console.WriteLine("Enter the specs JSON file path to be checked:");
+                var input = Console.ReadLine()?.Trim();
```

Comment:
For such tools, it's better to read arguments from the command line. This facilitates automation.
Let's also make sure that it can receive a specific file path, or a directory, from where the tool should load and validate each spec (one after the other).